src/js/entities-service/entities/events.js

```diff
@@ -33,8 +33,8 @@ const _Model = BaseModel.extend({
     return Radio.request('entities', 'programs:model', this.get('_program'));
   },
   getForm() {
-    if (!this.get('_form')) return;
-    return Radio.request('entities', 'forms:model', this.get('_form'));
+    if (!this.get('_form_revision')) return;
+    return Radio.request('entities', 'forms:model', this.get('_form_revision'));
```

Comment:
I'm not sure that swapping `form` with `form_revision` is going to be right in _all_ cases.

Reply:
it is here though isn't it?  (assuming it's the correct entity) because in the migration only program-action and workspace didn't get migrated.. plus this'd show whatever version of the form name went with that particular revision?

Reply:
Yes, this is fine. I am worried that mixing form/form-revision is going to lead to confusion/bugs down the road.